diff -urN -x CVS kdetoys.orig/debian/amor.1 kdetoys/debian/amor.1
--- kdetoys.orig/debian/amor.1	Sun Feb  1 08:14:38 2004
+++ kdetoys/debian/amor.1	Sat Oct 16 07:02:56 2004
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH AMOR 1 "October 13, 2003"
+.TH AMOR 1 "October 16, 2004"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -43,7 +43,8 @@
 \fIkdetoys-doc-html\fP and read this documentation in HTML format from
 \fI/usr/share/doc/kde/HTML/en/amor/\fP.
 .SH AUTHOR
-AMOR was written by Martin R. Jones <mjones@kde.org>.
+AMOR was written by Martin R. Jones <mjones@kde.org> and
+Gerardo Puga <gpuga@gioia.ing.unlp.edu.ar>.
 .br
 This manual page was prepared by Ben Burton <bab@debian.org>
 for the Debian GNU/Linux system (but may be used by others).
diff -urN -x CVS kdetoys.orig/debian/amor.docs kdetoys/debian/amor.docs
--- kdetoys.orig/debian/amor.docs	Thu Jan  1 01:00:00 1970
+++ kdetoys/debian/amor.docs	Sat Oct 16 07:02:56 2004
@@ -0,0 +1 @@
+amor/TODO
diff -urN -x CVS kdetoys.orig/debian/changelog kdetoys/debian/changelog
--- kdetoys.orig/debian/changelog	Sun Aug  1 10:43:34 2004
+++ kdetoys/debian/changelog	Sat Oct 16 07:02:56 2004
@@ -1,8 +1,10 @@
-kdetoys (4:3.2.92-1) experimental; urgency=low
+kdetoys (4:3.3.0-1) unstable; urgency=low
 
   * New upstream release.
+  * Fixed incorrect kweather sunset time (closes: #243840).
+  * Mention in the docs that kworldclock can be used as a desktop background.
 
- -- Ben Burton <bab@debian.org>  Sun,  1 Aug 2004 18:41:10 +1000
+ -- Ben Burton <bab@debian.org>  Sat, 16 Oct 2004 14:09:26 +1000
 
 kdetoys (4:3.2.3-3) unstable; urgency=low
 
diff -urN -x CVS kdetoys.orig/debian/control kdetoys/debian/control
--- kdetoys.orig/debian/control	Sun Aug  1 12:03:48 2004
+++ kdetoys/debian/control	Sat Oct 16 07:02:56 2004
@@ -2,7 +2,7 @@
 Section: kde
 Priority: optional
 Maintainer: Ben Burton <bab@debian.org>
-Build-Depends: automake1.7, debhelper (>> 4.0.0), kdelibs4-dev (>> 4:3.2.92)
+Build-Depends: automake1.7, debhelper (>> 4.0.0), kdelibs4-dev (>> 4:3.3.0)
 Standards-Version: 3.6.1
 
 Package: kdetoys
@@ -141,8 +141,8 @@
 Suggests: khelpcenter, kdetoys-doc-html
 Description: weather display applet for KDE
  An applet for the KDE panel that displays your area's current weather.
- Information shown includes the temperature, wind speed and air pressure.
- By pressing a button a full weather report can be obtained.
+ Information shown includes the temperature, wind speed, air pressure
+ and more.  By pressing a button a full weather report can be obtained.
  .
  KWeather also provides a weather service that can track multiple weather
  stations and provide this information to other applications, including
@@ -154,14 +154,15 @@
 Architecture: any
 Section: games
 Depends: ${shlibs:Depends}
-Suggests: kdeartwork-misc, kicker, khelpcenter, kdetoys-doc-html
+Suggests: kdeartwork-misc, kicker, kdesktop, khelpcenter, kdetoys-doc-html
 Conflicts: task-kdetoys, kworldwatch (<= 4:2.2-beta1-1)
 Replaces: kdetoys (<< 4:3.0.1-0), task-kdetoys, kworldwatch (<= 4:2.2-beta1-1)
 Provides: kworldwatch
 Description: earth watcher for KDE
  Displays where in the world it is light and dark depending on time, as
  well as offering the time in all of the major cities of the world.
- This can be run standalone or as an applet in the KDE panel.
+ This can be run standalone, as an applet in the KDE panel or as a
+ desktop background.
  .
  Additional kworldclock themes are available in the kdeartwork-misc package.
  .
diff -urN -x CVS kdetoys.orig/debian/copyright kdetoys/debian/copyright
--- kdetoys.orig/debian/copyright	Sun Feb  1 08:14:38 2004
+++ kdetoys/debian/copyright	Sat Oct 16 07:02:56 2004
@@ -9,29 +9,33 @@
 
 Upstream Authors:
 
-  AMOR: Martin R. Jones <mjones@kde.org>
+  AMOR: Martin R. Jones <mjones@kde.org> and
+      Gerardo Puga <gpuga@gioia.ing.unlp.edu.ar>
   Eyes Applet: Matthias Elter <elter@kde.org> and
-               Jerome Tollet <tollet@magic.fr>
+      Jerome Tollet <tollet@magic.fr>
   Fifteen Pieces Applet: Matthias Elter <elter@kde.org>
   KMoon: Stephan Kulow <coolo@kde.org>,
-         Christopher Osburn <chris@speakeasy.org>
-         and Tim Beauchamp <timb@googol.com>
-  KOdometer: Armen Nakashian <armen@tourismo.com>
+      M G berberich <berberic@fmi.uni-passau.de>,
+      Christopher Osburn <chris@speakeasy.org>
+      and Tim Beauchamp <timb@googol.com>
+  KOdometer: Armen Nakashian <armen@tourismo.com> and
+      Mark H. Granoff/mhg <granoff@keptin.lkg.dec.com>
   KTeaTime: Matthias Hoelzer-Kluepfel <hoelzer@kde.org> and
-            Martin Willers <willers@xm-arts.de>
+      Martin Willers <willers@xm-arts.de>
   KTux: Martin R. Jones <mjones@kde.org> and Carlos Rego
   KWeather: Ian Reinhart Geiser <geiseri@kde.org>,
-            Nadeem Hasan <nhasan@kde.org> and
-            Tobias Koenig <tokoe@kde.org>
-  KWorldClock: Matthias Hoelzer-Kluepfel <hoelzer@kde.org>
+      Nadeem Hasan <nhasan@kde.org> and Tobias Koenig <tokoe@kde.org>
+  KWorldClock: Matthias Hoelzer-Kluepfel <hoelzer@kde.org> and others
 
 Copyright:
 
   All components of this package are released under the GNU General
   Public License, with the exceptions shown below.
 
-  On Debian GNU/Linux systems, the complete text of the GNU General
-  Public License can be found in `/usr/share/common-licenses/GPL'.
+  On Debian GNU/Linux systems, the complete texts of the GNU General
+  Public License (GPL) and the GNU Library General Public License (LGPL)
+  can be found in the files `/usr/share/common-licenses/GPL' and
+  `/usr/share/common-licenses/LGPL'.
 
   --------------------
 
@@ -87,6 +91,13 @@
 
   --------------------
 
+  KWeather:
+
+  Primarily under the GPL (GNU General Public License), but some sources
+  are under the LGPL (GNU Library General Public License).
+
+  --------------------
+
   KWorldClock:
 
   GPL (GNU General Public License) with the following exception:
diff -urN -x CVS kdetoys.orig/debian/kdetoys-doc-html.doc-base.kweather kdetoys/debian/kdetoys-doc-html.doc-base.kweather
--- kdetoys.orig/debian/kdetoys-doc-html.doc-base.kweather	Sun Feb  1 08:14:38 2004
+++ kdetoys/debian/kdetoys-doc-html.doc-base.kweather	Sat Oct 16 07:02:56 2004
@@ -1,6 +1,8 @@
 Document: kdetoys-doc-html-kweather
 Title: KWeather Handbook
-Abstract: KWeather is an applet that can display your area's current weather.
+Abstract: KWeather is an application that can display your area's current
+ weather.  It provides a panel applet as well as a weather service that
+ can supply information to other applications.
 Section: Apps/Tools
 
 Format: HTML
diff -urN -x CVS kdetoys.orig/debian/kdetoys-doc-html.doc-base.kworldclock kdetoys/debian/kdetoys-doc-html.doc-base.kworldclock
--- kdetoys.orig/debian/kdetoys-doc-html.doc-base.kworldclock	Thu Jan  1 01:00:00 1970
+++ kdetoys/debian/kdetoys-doc-html.doc-base.kworldclock	Sat Oct 16 07:02:56 2004
@@ -0,0 +1,12 @@
+Document: kdetoys-doc-html-kworldclock
+Title: KWorldClock Handbook
+Author: Brad Hards
+Abstract: KWorldClock is an application that shows which parts of the world
+ are currently experiencing daylight, and which parts are currently in night.
+ It also shows the current time in a range of cities around the world.
+Section: Apps/Games
+
+Format: HTML
+Index: /usr/share/doc/kde/HTML/en/kworldclock/index.html
+Files: /usr/share/doc/kde/HTML/en/kworldclock/*.html
+
diff -urN -x CVS kdetoys.orig/debian/kmoon.install kdetoys/debian/kmoon.install
--- kdetoys.orig/debian/kmoon.install	Sun Aug  1 12:03:48 2004
+++ kdetoys/debian/kmoon.install	Sat Oct 16 07:02:56 2004
@@ -10,4 +10,3 @@
 usr/share/icons/crystalsvg/48x48/apps/kmoon.png
 usr/share/icons/crystalsvg/32x32/apps/kmoon.png
 usr/share/icons/crystalsvg/16x16/apps/kmoon.png
-../kmoon.xpm usr/share/pixmaps
diff -urN -x CVS kdetoys.orig/debian/kodo.1 kdetoys/debian/kodo.1
--- kdetoys.orig/debian/kodo.1	Sun Feb  1 08:14:38 2004
+++ kdetoys/debian/kodo.1	Sat Oct 16 07:02:56 2004
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH KODO 1 "May 12, 2003"
+.TH KODO 1 "October 16, 2004"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -41,7 +41,8 @@
 \fIkdetoys-doc-html\fP and read this documentation in HTML format from
 \fI/usr/share/doc/kde/HTML/en/kodo/\fP.
 .SH AUTHOR
-KOdometer was written by Armen Nakashian <armen@tourismo.com>.
+KOdometer was written by Armen Nakashian <armen@tourismo.com> and
+Mark H. Granoff/mhg <granoff@keptin.lkg.dec.com>.
 .br
 This manual page was prepared by Ben Burton <bab@debian.org>
 for the Debian GNU/Linux system (but may be used by others).
diff -urN -x CVS kdetoys.orig/debian/kweatherreport.1 kdetoys/debian/kweatherreport.1
--- kdetoys.orig/debian/kweatherreport.1	Sun Feb  1 08:14:38 2004
+++ kdetoys/debian/kweatherreport.1	Sat Oct 16 07:02:56 2004
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH KWEATHERREPORT 1 "January 31, 2004"
+.TH KWEATHERREPORT 1 "October 16, 2004"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -22,8 +22,8 @@
 .RI "[ " generic-options " ] " location
 .SH DESCRIPTION
 \fBkweatherreport\fP is a small program that asks KWeatherService to display
-a weather report.  Information shown includes the temperature, wind speed
-and air pressure.
+a weather report.  Information shown includes the temperature, wind speed,
+air pressure and more.
 .PP
 \fBKWeatherService\fP is a DCOP service that can track multiple weather
 stations to provide weather data.  There is no need to start
diff -urN -x CVS kdetoys.orig/debian/kworldclock.1 kdetoys/debian/kworldclock.1
--- kdetoys.orig/debian/kworldclock.1	Sat Sep  6 18:18:41 2003
+++ kdetoys/debian/kworldclock.1	Sat Oct 16 07:02:56 2004
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH KWORLDCLOCK 1 "May 12, 2003"
+.TH KWORLDCLOCK 1 "October 16, 2004"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -24,11 +24,13 @@
 [ \fB\-\-theme\fP \fIfile\fP ]
 [ \fB\-o\fP \fIfile\fP ]
 [ \fB\-\-size\fP\ \fIW\fP\fBx\fP\fIH\fP ]]
+.PP
+.B kworldclock \-\-themes
 .SH DESCRIPTION
 \fBKWorldClock\fP displays where in the world it is light and dark
 depending on time, as well as offering the time in all of the major
-cities of the world.  This can be run standalone or as an applet in the
-KDE panel.
+cities of the world.  This can be run standalone, as an applet in the
+KDE panel or as a desktop background.
 .PP
 This program is part of the official KDE toys module.
 .SH OPTIONS
@@ -39,15 +41,30 @@
 Write out a file containing the actual map.
 .TP
 \fB\-\-theme\fP \fIfile\fP
-The name of the theme to use for a map dump (default: earth).
+The name of the theme to use for a map dump (default: depths).
+.TP
+\fB\-\-themes\fP
+Display a list of all available themes.
 .TP
 \fB\-o\fP \fIfile\fP
 The name of the file to write to for a map dump (default: dump.ppm).
 .TP
 \fB\-\-size\fP\ \fIW\fP\fBx\fP\fIH\fP
 The size of the map to dump.
+.SH SEE ALSO
+Full user documentation is available through the KDE Help Centre.
+You can also enter the URL
+\fIhelp:/kworldclock/\fP
+directly into konqueror or you can run
+`\fIkhelpcenter help:/kworldclock/\fP'
+from the command-line.
+.PP
+If the KDE Help Centre is not installed then you can install the package
+\fIkdetoys-doc-html\fP and read this documentation in HTML format from
+\fI/usr/share/doc/kde/HTML/en/kworldclock/\fP.
 .SH AUTHOR
-KWorldClock was written by Matthias Hoelzer-Kluepfel <hoelzer@kde.org>.
+KWorldClock was written by Matthias Hoelzer-Kluepfel <hoelzer@kde.org>
+and others.
 .br
 This manual page was prepared by Ben Burton <bab@debian.org>
 for the Debian GNU/Linux system (but may be used by others).
diff -urN -x CVS kdetoys.orig/debian/kworldclock.README.Debian kdetoys/debian/kworldclock.README.Debian
--- kdetoys.orig/debian/kworldclock.README.Debian	Thu Jan  1 01:00:00 1970
+++ kdetoys/debian/kworldclock.README.Debian	Sat Oct 16 07:02:56 2004
@@ -0,0 +1,8 @@
+KWorldClock for Debian
+----------------------
+
+This application provides documentation in a docbook format that the KDE
+Help Centre can display.  For HTML documentation that can be read in any
+web browser, see the package kdetoys-doc-html.
+
+ -- Ben Burton <bab@debian.org>, Sat, 16 Oct 2004 14:09:26 +1000
diff -urN -x CVS kdetoys.orig/debian/source.lintian-overrides kdetoys/debian/source.lintian-overrides
--- kdetoys.orig/debian/source.lintian-overrides	Sun Aug  1 10:29:47 2004
+++ kdetoys/debian/source.lintian-overrides	Wed Oct 13 03:35:18 2004
@@ -1,2 +1,2 @@
-kdetoys source: cvsignore-file-in-source
-kdetoys source: source-contains-CVS-dir
+cvsignore-file-in-source
+source-contains-CVS-dir
diff -urN -x CVS kdetoys.orig/kmoon/kmoonapplet.desktop kdetoys/kmoon/kmoonapplet.desktop
--- kdetoys.orig/kmoon/kmoonapplet.desktop	Sun Sep 19 16:03:52 2004
+++ kdetoys/kmoon/kmoonapplet.desktop	Mon Oct  4 15:26:51 2004
@@ -29,7 +29,7 @@
 Name[sk]=Fázy Mesiaca
 Name[sl]=Prikaz Luninih men
 Name[sr]=Приказ месечевих мена
-Name[sr@Latn]=Приказ месечевих мена
+Name[sr@Latn]=Prikaz mesečevih mena
 Name[sv]=Visning av månfaser
 Name[ta]=சந்திரன் மாறுபாட்டு நிலை காட்சி
 Name[tg]=Намоиши Марҳилаҳои Моҳ
@@ -62,7 +62,7 @@
 Comment[sk]=Applet pre Fázy mesiaca
 Comment[sl]=Vstavek za Lunine mene v KDE
 Comment[sr]=Апет који приказује месечеве мене
-Comment[sr@Latn]=Апет који приказује месечеве мене
+Comment[sr@Latn]=Apet koji prikazuje mesečeve mene
 Comment[sv]=Ett KDE-miniprogram för månfaser
 Comment[ta]= A KDE சந்திரன் மாறுபாட்டு நிலை குறுநிரல்
 Comment[tg]=KDE апплети марҳилаҳои Моҳ
diff -urN -x CVS kdetoys.orig/ktux/ktux.desktop kdetoys/ktux/ktux.desktop
--- kdetoys.orig/ktux/ktux.desktop	Thu Sep 23 16:01:48 2004
+++ kdetoys/ktux/ktux.desktop	Mon Oct  4 15:26:53 2004
@@ -121,7 +121,7 @@
 Name[sk]=Zobraziť v zadanom okne
 Name[sl]=Prikaz v določenemu oknu
 Name[sr]=Прикажи у задатом прозору
-Name[sr@Latn]=Прикажи у задатом прозору
+Name[sr@Latn]=Prikaži u zadatom prozoru
 Name[sv]=Visa i angivet fönster
 Name[ta]= குறிப்பிட்ட சாளர காட்சியை காட்டு
 Name[tg]=Дар Тирезаи Муайяншуда Намоиш додан
@@ -159,7 +159,7 @@
 Name[sk]=Zobraziť v koreňovom okne
 Name[sl]=Prikaz v korenskem oknu
 Name[sr]=Прикажи у кореном прозору
-Name[sr@Latn]=Прикажи у кореном прозору
+Name[sr@Latn]=Prikaži u korenom prozoru
 Name[sv]=Visa i rotfönster
 Name[ta]=மூல சாளரத்தில் காட்டு
 Name[tg]=Дар Тирезаи Решавӣ Намоиш додан
diff -urN -x CVS kdetoys.orig/kworldwatch/mapwidget.cpp kdetoys/kworldwatch/mapwidget.cpp
--- kdetoys.orig/kworldwatch/mapwidget.cpp	Fri Jul  9 11:30:12 2004
+++ kdetoys/kworldwatch/mapwidget.cpp	Tue Oct 12 01:26:11 2004
@@ -174,13 +174,16 @@
   setIllumination(config->readBoolEntry("Illumination", true));
   setFlags(config->readBoolEntry("Flags", true));
 
-  KCmdLineArgs *args = KCmdLineArgs::parsedArgs();
-  if (args->isSet("theme"))
-    setTheme(QString::fromLocal8Bit(args->getOption("theme")));
-  else
-    setTheme(config->readEntry("Theme", "depths"));
+  if ( ! _applet )
+  {
+    KCmdLineArgs *args = KCmdLineArgs::parsedArgs();
+    if (args->isSet("theme"))
+      setTheme(QString::fromLocal8Bit(args->getOption("theme")));
+    else
+      setTheme(config->readEntry("Theme", "depths"));
 
-  _flagList->load(config);  
+    _flagList->load(config);
+  }
 }
 
 
