Index: kworldwatch/maploader.cpp
===================================================================
--- kworldwatch/maploader.cpp	(.../tags/KDE/3.5.10/kdetoys)	(revision 958024)
+++ kworldwatch/maploader.cpp	(.../branches/KDE/3.5/kdetoys)	(revision 958024)
@@ -88,11 +88,11 @@
   // find the closest (bigger) size
   uint size=0;
   for (uint i=0; i<sizes.count(); ++i)
-    if (sizes[i] >= width)
-      {
-	size = sizes[i];
+    {
+      size = sizes[i];
+      if (size >= width)
 	break;
-      }
+    }
   
   QImage image;
   if (size == 0)

Property changes on: .
___________________________________________________________________
Deleted: svn:mergeinfo
Added: svn:externals
   + admin https://svn.kde.org/home/kde/branches/KDE/3.5/kde-common/admin


