--- kdetoys-3.0.3/eyesapplet/Makefile.am.orig	Fri Mar  8 02:19:08 2002
+++ kdetoys-3.0.3/eyesapplet/Makefile.am	Wed Sep 25 00:13:28 2002
@@ -12,7 +12,7 @@
 
 EXTRA_DIST = $(lnk_DATA)
 
-eyes_panelapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined
+eyes_panelapplet_la_LDFLAGS = $(all_libraries) -module -no-undefined -avoid-version
 eyes_panelapplet_la_LIBADD = $(LIB_KDEUI)
 
 messages:
--- kdetoys-3.0.3/eyesapplet/Makefile.in.orig	Wed Aug  7 20:27:23 2002
+++ kdetoys-3.0.3/eyesapplet/Makefile.in	Wed Sep 25 00:13:57 2002
@@ -240,7 +240,7 @@
 
 EXTRA_DIST = $(lnk_DATA)
 
-eyes_panelapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined
+eyes_panelapplet_la_LDFLAGS = $(all_libraries) -module -no-undefined -avoid-version
 eyes_panelapplet_la_LIBADD = $(LIB_KDEUI)
 subdir = eyesapplet
 mkinstalldirs = $(SHELL) $(top_srcdir)/admin/mkinstalldirs
--- kdetoys-3.0.3/fifteenapplet/Makefile.am.orig	Fri Mar  8 02:19:08 2002
+++ kdetoys-3.0.3/fifteenapplet/Makefile.am	Wed Sep 25 00:16:02 2002
@@ -12,7 +12,7 @@
 
 EXTRA_DIST = $(lnk_DATA)
 
-fifteen_panelapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined
+fifteen_panelapplet_la_LDFLAGS = $(all_libraries) -module -no-undefined -avoid-version
 fifteen_panelapplet_la_LIBADD = $(LIB_KDEUI)
 
 messages:
--- kdetoys-3.0.3/fifteenapplet/Makefile.in.orig	Wed Aug  7 20:27:23 2002
+++ kdetoys-3.0.3/fifteenapplet/Makefile.in	Wed Sep 25 00:16:29 2002
@@ -240,7 +240,7 @@
 
 EXTRA_DIST = $(lnk_DATA)
 
-fifteen_panelapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined
+fifteen_panelapplet_la_LDFLAGS = $(all_libraries) -module -no-undefined -avoid-version
 fifteen_panelapplet_la_LIBADD = $(LIB_KDEUI)
 subdir = fifteenapplet
 mkinstalldirs = $(SHELL) $(top_srcdir)/admin/mkinstalldirs
--- kdetoys-3.0.3/kweather/Makefile.am.orig	Fri Mar  8 02:19:08 2002
+++ kdetoys-3.0.3/kweather/Makefile.am	Tue Sep 24 23:26:45 2002
@@ -13,7 +13,7 @@
 datadir = $(kde_datadir)/kicker/applets/
 data_DATA = kweather.desktop
 
-weather_panelapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined
+weather_panelapplet_la_LDFLAGS = $(all_libraries) -module -no-undefined -avoid-version
 weather_panelapplet_la_LIBADD = $(LIB_KFILE) $(LIB_KDEUI) $(LIB_KHTML)
 
 
--- kdetoys-3.0.3/kweather/Makefile.in.orig	Wed Aug  7 20:27:23 2002
+++ kdetoys-3.0.3/kweather/Makefile.in	Tue Sep 24 23:27:29 2002
@@ -242,7 +242,7 @@
 noinst_HEADERS = dockwidget.h kweather.h prefsDialog.h prefdialogview.h weatherlib.h weatherIface.h  reportview.h
 data_DATA = kweather.desktop
 
-weather_panelapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined
+weather_panelapplet_la_LDFLAGS = $(all_libraries) -module -no-undefined -avoid-version
 weather_panelapplet_la_LIBADD = $(LIB_KFILE) $(LIB_KDEUI) $(LIB_KHTML)
 subdir = kweather
 mkinstalldirs = $(SHELL) $(top_srcdir)/admin/mkinstalldirs
--- kdetoys-3.0.3/kworldwatch/Makefile.am.orig	Fri Mar  8 02:19:10 2002
+++ kdetoys-3.0.3/kworldwatch/Makefile.am	Wed Sep 25 00:41:47 2002
@@ -36,7 +36,7 @@
 
 EXTRA_DIST = $(lnk_DATA)
 
-ww_panelapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined $(KDE_RPATH)
+ww_panelapplet_la_LDFLAGS = $(all_libraries) -module -no-undefined -avoid-version $(KDE_RPATH)
 ww_panelapplet_la_LIBADD = libww.la $(LIB_KSYCOCA) 
 
 messages: rc.cpp
--- kdetoys-3.0.3/kworldwatch/Makefile.in.orig	Wed Aug  7 20:27:23 2002
+++ kdetoys-3.0.3/kworldwatch/Makefile.in	Wed Sep 25 00:42:24 2002
@@ -266,7 +266,7 @@
 
 EXTRA_DIST = $(lnk_DATA)
 
-ww_panelapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined $(KDE_RPATH)
+ww_panelapplet_la_LDFLAGS = $(all_libraries) -module -no-undefined -avoid-version $(KDE_RPATH)
 ww_panelapplet_la_LIBADD = libww.la $(LIB_KSYCOCA) 
 subdir = kworldwatch
 mkinstalldirs = $(SHELL) $(top_srcdir)/admin/mkinstalldirs
